webapp/src/api.rs

```diff
@@ -8,6 +8,7 @@ use axum::Json;
 use native::api;
 use native::api::Fee;
 use native::api::SendPayment;
+use native::api::WalletHistoryItemType;
```

Comment:
🤔 I thought the idea was that we would reuse everything in `native` _except_ for the stuff in the `api` modules.

Reply:
I fear the models are only on the `api` layer.

Reply:
Right! I think it might pay off to fix that, because we probably don't want to change the web app models when we are working on the mobile app.

For now we could just duplicate a couple of times.

Reply:
I am not so sure about that. In the past we've copied the models extensively for the different modules (common, database, api, etc) and it was very cumbersome to write all this boilerplate code to copy data from one model to the other (which was mostly exactly the same). 

For now I suggest we keep it as is and revisit if we run into any issues with it. Note, the api models used between webapp ui and webapp backend are decoupled from the native crate.

Reply:
It would at least be sensible to move the shared code to a place that is not explicitly designed to be consumed by `flutter_rust_bridge` though.

Reply:
ok, I am going to duplicate the code then 🙈